__tests__/commands/install/integration.js

```diff
@@ -809,6 +810,36 @@ test.concurrent('lockfile should be created when missing even if integrity match
   });
 });
 
+test.concurrent('offline mirror can be enabled from parent dir', (): Promise<void> => {
+  return runInstall({}, 'offline-mirror-configuration/enabled-from-parent', async (config, reporter) => {
+    const rawLockfile = await fs.readFile(path.join(config.cwd, 'yarn.lock'));
+    const lockfile = parse(rawLockfile);
+    expect(lockfile['mime-types@2.1.14'].resolved).toEqual(
+      'mime-types-2.1.14.tgz#f7ef7d97583fcaf3b7d282b6f8b5679dab1e94ee',
```

Comment:
Oh, https://github.com/yarnpkg/yarn/pull/2970 landed just yesterday, we don't strip `https://registry.yarnpkg.com/mime-types/-/` from the lockfile anymore if we are using offline mirror.

You probably can check that file was added to the offline mirror folder as an indicator.